Loader/ComboLoader.php

```diff
@@ -162,4 +178,31 @@ private function findFileExtension($file)
     {
         return substr(strrchr($file, '_'), 1);
     }
+
+    private function isTemplateFile($file)
+    {
+        return preg_match('#/tpl/handlebars/.*_js#', $file) === 1;
+    }
+
+    private function renderTemplate($file)
+    {
+        preg_match('#/tpl/handlebars/(.*)_js#', $file, $matches);
+        $module = $matches[1];
+
+        $path = $this->getModuleSetting($module, 'path');
+
+        return $this->templating->render(
```

Comment:
this is kind of duplicating https://github.com/ezsystems/PlatformUIBundle/blob/master/Controller/TemplateController.php#L42
Can't this be reused somehow ?